lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -106,6 +106,12 @@ sub remove_from_disk {
         log_info("GRU: skipping removal of $type/$name; it does not exist anyways");
         return undef;
     }
+    if (my $target = readlink $file) {
+        if ($target =~ qr{^fixed/} && -e $target) {
```

Comment:
This would of course only work for relative symlinks but looking at the list of symlinks on OSD it would be good enough.